Branch-SDK/Branch-SDK/BNCPreferenceHelper.h

```diff
@@ -31,8 +31,9 @@
 @property (assign, nonatomic) BOOL explicitlyRequestedReferrable;
 @property (assign, nonatomic) BOOL isReferrable;
 @property (assign, nonatomic) BOOL isDebug;
-@property (assign, nonatomic) BOOL isContinuingUserActivity;
+@property (assign, nonatomic) BOOL shouldWaitForInit;
```

Comment:
How about `initWillBeCalledSoon`? In your referenced code snippet, there's no reason for that boolean there. That boolean is in the wrong place. Sorry, I didn't review that code. Just changed the variable name. That should look like this:
```
if (self.accountForFacebookSDK) {
    id activity = [[options objectForKey:UIApplicationLaunchOptionsUserActivityDictionaryKey] objectForKey:@"UIApplicationLaunchOptionsUserActivityKey"];
    if (activity && [activity isKindOfClass:[NSUserActivity class]]) {
        return [self continueUserActivity:activity];
    }
}
self.preferenceHelper.shouldWaitForInit = YES;```